samples/drivers/usbc/sink/boards/stm32g071b_disco.overlay

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022 The Chromium OS Authors.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#include <dt-bindings/sensor/ina230.h>
+
+/ {
+	aliases {
+		tcpc = &ucpd1;
+		vbus = &vbus_meas;
+	};
+
+	cc_config {
+		compatible = "gpio-leds";
+		encc1 {
+			gpios = <&gpiob 10 GPIO_ACTIVE_HIGH>;
+			label = "ENCC1";
+		};
+
+		encc2 {
+			gpios = <&gpiob 11 GPIO_ACTIVE_HIGH>;
+			label = "ENCC2";
+		};
+
+		rdcc1 {
+			gpios = <&gpiob 12 GPIO_ACTIVE_LOW>;
+			label = "RDCC1";
+		};
+	};
+};
+
+&ucpd1 {
+	status = "okay";
+
+	/*
+	 * UCPD is fed directly from HSI which is @ 16MHz. The ucpd_clk goes to
+	 * a prescaler who's output feeds the 'half-bit' divider which is used
+	 * to generate clock for delay counters and BMC Rx/Tx blocks. The rx is
+	 * designed to work in freq ranges of 6 <--> 18 MHz, however recommended
+	 * range is 9 <--> 18 MHz.
+	 *
+	 *          ------- @ 16 MHz ---------   @ ~600 kHz   -------------
+	 * HSI ---->| /psc |-------->| /hbit |--------------->| trans_cnt |
+	 *          -------          ---------    |           -------------
+	 *                                        |           -------------
+	 *                                        |---------->| ifrgap_cnt|
+	 *                                                    -------------
+	 * Requirements:
+	 *   1. hbit_clk ~= 600 kHz: 16 MHz / 600 kHz = 26.67
+	 *   2. tTransitionWindow - 12 to 20 uSec
+	 *   3. tInterframGap - uSec
+	 *
+	 * hbit_clk = HSI_clk / 27 = 592.6 kHz = 1.687 uSec period
+	 * tTransitionWindow = 1.687 uS * 8 = 13.5 uS
+	 * tInterFrameGap = 1.687 uS * 17 = 28.68 uS
+	 */
+	psc-ucpdclk = <1>;
+	hbitclkdiv = <27>;
+};
+
+&i2c1 {
+	pinctrl-0 = <&i2c1_scl_pb6 &i2c1_sda_pb7>;
+	pinctrl-names = "default";
+	status = "okay";
```

Comment:
if board always uses these pins for I2C, this part should be moved there.

Reply:
Done